lms/static/scripts/frontend_apps/services/client-rpc.js

```diff
@@ -131,17 +168,35 @@ export class ClientRPC extends TinyEmitter {
    *  focused user's groups
    */
   async setFocusedUser(user, groups) {
+    await this._callClient('changeFocusModeUser', {
+      // Passing `undefined` as the `username` disables focus mode in the client.
+      //
+      // TODO: The `username` property is deprecated in the client and should be
+      // changed to `userid` once the client no longer references `username`.
+      username: user ? user.userid : undefined,
+      displayName: user ? user.displayName : undefined,
+      groups: groups ?? undefined,
+    });
+  }
+
+  /**
+   * Show the content information banner above the content the user is
+   * annotating.
```

Comment:
I have revised this description to make it clearer that the LMS app is instructing the client to show the banner, rather than showing the banner itself.